ChallengeOneWeek2.html

```diff
@@ -1,19 +1,28 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <script>
-      var birthyear = prompt("enter year of birth");
-      var currentyear = new Date().getFullYear();
-      var yeardifference = currentyear - birthyear;
-      if (yeardifference < 18) {
-        console.log("You are a minor");
-      } else if (18 <= yeardifference && yeardifference <= 36) {
-        console.log("You are a Youth");
-      } else {
-        console.log("you are an Elder");
-      }
-    </script>
-  </head>
-
-  <body></body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head> </head>
+
+  <body>
+    <script>
+      while (true) {
+        var birthyear = prompt("enter year of birth");

```

Comment:
Use `camelCase` when naming your variables i.e `birthYear` instead of `birthyear`.

**TLDR;**
Usually, language communities have come up with conventions of naming variables. In the JavaScript community and codebases, `camelCase` is commonly used for naming variables and functions, whereas `PascalCase` is commonly used for naming classes. This is usually done for readability because code is more read than written. The code you write today will be read by someone else on your team tomorrow. For example `aVeryLongVariableName` is easy to read compared to `averylongvariablename`.

Reply:
added camelCase. thanks for the correction